docs/sources/developers/plugins/add-authentication-for-data-source-plugins.md

```diff
@@ -154,3 +154,31 @@ Grafana automatically renews the token when it expires.
   }
 ]
 ```
+
+## Optional headers to retrieve OAuth token
+
+If a header value is expected in the call to the oauth url, they can sent using
```

Comment:
```suggestion
If the call to the OAuth URL expects a header value, you can send it using
```

Prefer active over passive voice.

Reply:
Also, I think this belongs under the "Enable token authentication" heading rather as a top-level heading.